types-core/index.d.ts

```diff
@@ -43,6 +43,11 @@ declare module '@azure/functions-core' {
          * A dictionary of binding name to binding info
          */
         bindings: { [name: string]: RpcBindingInfo };
+
+        /**
+         * The retry policy options
+         */
+        retryOptions?: RpcRetryOptions;
```

Comment:
Why is this in the `FunctionMetadata` interface if it's not required? Unless I understand the distinction between `FunctionMetadata` and `RpcFunctionMetadata` incorrectly

Reply:
I don't entirely remember why they're split up, but `FunctionMetadata` is what's used in the `registerFunction` api so it needs a property for anything the library package might specify when calling `registerFunction`

`RpcFunctionMetadata` is basically a strict copy of the generated rpc types and isn't cleaned up in any way.

Reply:
I'm basing my comment here based on the doc string for this type: "A slimmed down version of `RpcFunctionMetadata` that includes the _minimum amount_ of information needed to register a function." (emphasis added). Maybe this doc string should be updated to be more accurate?

Reply:
sure - messed with the comment a bit